repository-data/webfiles/src/main/resources/site/freemarker/common/facets.ftl

```diff
@@ -32,15 +32,20 @@
             <h3 class="filter-list-title"><@fmt.message key=facet.name /></h3>
             <ul class="filter-list" title="<@fmt.message key=facet.name />" data-max-count="${facetMaxCount}" data-state="short">
                 <#list facetItems as value>
+                    <#assign valueName="Not Defined"/>
                     <#if facet.name="month">
                         <@fmt.message key=value.name var="monthName"/>
                         <#assign valueName=monthName/>
                     <#elseif facet.name="category">
                         <#assign valueName=taxonomy.getValueName(value.name)/>
                     <#elseif facet.name="document-type">
-                        <@fmt.message key="facet."+value.name var="valueName"/>  
+                        <@fmt.message key="facet."+value.name var="docType"/>
+                        <#assign valueName=docType/>
```

Comment:
I didn't get it :( Why 2 assignments?

Reply:
For some reason the `<@fmt.message key="facet."+value.name var="docType"/>` variable appears to be somewhat final in that you can't reassign it in the second facet type. Where as if you call `#assign` you can reassign the variable. Without this the second use of `@fmt.message` doesn't work and you just get the previous value repeated. Weird!